packages/hw-app-ada/src/Ada.js

```diff
@@ -53,15 +53,15 @@ export default class Ada {
   transport: Transport<*>;
   methods: Array<string>;
 
-  constructor(transport: Transport<*>) {
+  constructor(transport: Transport<*>, scrambleKey: ?string) {
```

Comment:
I suggest to move to default param syntax:

```
scrambleKey: string = "ADA"
```

that way you can directly pass `scrambleKey` down (and it's not an option since it get defaulted)

Reply:
good call 👍